pkg/controller/drain.go

```diff
@@ -603,21 +603,21 @@ func (o *DrainOptions) evictPodsWithPVInternal(attemptEvict bool, pods []*corev1
 
 		if attemptEvict && apierrors.IsTooManyRequests(err) {
 			// Pod eviction failed because of PDB violation, we will retry one we are done with this list.
-			glog.V(3).Info("Pod ", pod.Namespace, "/", pod.Name, " couldn't be evicted because of PDB violation. Will be retried.")
+			glog.V(3).Info("Pod ", pod.Namespace, "/", pod.Name, " from node ", pod.Spec.NodeName, " couldn't be evicted. This may also occur due to PDB violation. Will be retried. Error:", err)
 			retryPods = append(retryPods, pod)
 			continue
 		} else if apierrors.IsNotFound(err) {
-			glog.V(3).Info("\t", pod.Name, " is already gone")
+			glog.V(3).Info("\t", pod.Name, " from node ", pod.Spec.NodeName, " is already gone")
 			returnCh <- nil
 			continue
 		} else if err != nil {
-			glog.V(4).Infof("Error when evicting pod: %v/%v. Will be retried. Err: %v", pod.Namespace, pod.Name, err)
+			glog.V(4).Infof("Error when evicting pod: %v/%v from node %v. Will be retried. Err: %v", pod.Namespace, pod.Name, pod.Spec.NodeName, err)
 			retryPods = append(retryPods, pod)
 			continue
 		}
 
 		// Eviction was successful. Wait for pvs for this pod to detach
-		glog.V(3).Info("Eviction/delete succeeded. Waiting for the volumes to detach: ", pod.Name)
+		glog.V(3).Info("Eviction/delete succeeded. Waiting for the volumes to detach: ", pod.Name, pod.Spec.NodeName)
```

Comment:
We need to capture the time taken for eviction/deletion of the pod here. We might have to introduce a time variable for the same.